app/celery/tasks.py

```diff
@@ -192,15 +195,17 @@ def process_rows(rows: List, template: Template, job: Job, service: Service):
     # the same_sms and save_email task are going to be using template and service objects from cache
     # these objects are transient and will not have relationships loaded
     if encrypted_smss:
-        save_smss.apply_async(
+        chain(save_smss.s(
             (str(service.id), encrypted_smss, None),
-            queue=choose_database_queue(str(template.process_type), service.research_mode, job.notification_count),
-        )
+            queue=choose_database_queue(str(template.process_type), service.research_mode, job.notification_count)),
+            job_complete.s(job.id)
```

Comment:
`process_rows` will get called multiple times per job for larger jobs, once on each chunk of 500 rows. So we can't call the job complete after 1 chunk has finished processing.